addons/account_peppol/models/account_move.py

```diff
@@ -22,14 +22,37 @@ class AccountMove(models.Model):
         string='PEPPOL status',
         copy=False,
     )
+    move_sent_values = fields.Selection(
+        selection_add=[
+            ('peppol_pending', 'PEPPOL Pending'),
+            ('peppol_error', 'PEPPOL Error'),
+        ],
+    )
+
+    @api.depends('peppol_move_state', 'sending_data')
+    def compute_move_sent_values(self):
+        # EXTENDS 'account'
+        super().compute_move_sent_values()
+        for move in self:
+            # if the move was exclusively sent over peppol, we can use peppol-specific sent labels
+            if (move.sending_data or {}).get('sending_methods') == ['peppol']:
+                if move.peppol_move_state == 'processing':
+                    move.move_sent_values = 'peppol_pending'
+                    move.move_sent_values_decoration = 'none'
+                elif move.peppol_move_state in ('error', 'skipped'):
+                    move.move_sent_values = 'peppol_error'
+                    move.move_sent_values_decoration = 'danger'
```

Comment:
It's a bit sousoupe.
I'm wondering if we shouldn't take the chance to clean that up.
At the moment `is_move_sent` is actually set to True when the PDF is generated, regardless of whether it's sent by Email or any other mean. TBD